ied-be/src/utils/seminariQueryBuilder.ts

```diff
@@ -22,5 +25,26 @@ export function createSeminarQuery(params: FirmaQueryParams): FilterQuery<Semina
     query.predavac = { $regex: params.predavac, $options: "i" }; // Case-insensitive partial match
   }
 
+  if (params?.datum && params.datum.length > 0) {
```

Comment:
jasno mi je sta ovde pokusavas, ali ovde pretrazujes datum, ne deo datuma tako da nam to ne odgovara.
sledece ovde nije napomenuto u kom formatu se nalazi datum i kako cemo da ga uporedjujemo tako da ce to da ti bude sledeci zadatak.